ps_emailalerts.php

```diff
@@ -292,12 +292,17 @@ public function getAllMessages($id)
      *
      * @param Context $context
      *
-     * @return \PrestaShop\PrestaShop\Core\Localization\Locale
+     * @return \PrestaShop\PrestaShop\Core\Localization\Locale|null
      *
      * @throws Exception
      */
     public static function getContextLocale(Context $context)
     {
+        $locale = $context->getCurrentLocale();
+        if (null !== $locale) {
+            return $locale;
+        }
+
```

Comment:
I believe this whole function content can actually be replaced with:
```suggestion
        return Tools::getContextLocale($context);
```

Reply:
Ahhh... available as public since 177, so nevermind ^^